test/LeanCode.Test.Helpers/IntegrationFactAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using Xunit;
+using Xunit.v3;
+
+namespace LeanCode.Test.Helpers;
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+public class IntegrationFactAttribute : FactAttribute, ITraitAttribute
+{
+    public IntegrationFactAttribute()
+    {
+        Explicit = true;
+    }
+
+    public virtual IReadOnlyCollection<KeyValuePair<string, string>> GetTraits() =>
+        [new("category", "integration"), new("integration", "true")];
```

Comment:
Why two tags? Shouldn't `category=integration` suffice?

Reply:
Unfortunately, no.

The main culprit is the "explicit" tests. If not filtered out, they are reported as "pending." Thus, the CI reported the incorrect number of tests (because the explicit ones were counted as both "skipped in unit" and "executed in integration"). I didn't want to remove the explicit tag because it allows for a plain `dotnet test` run when running locally.

I went with an additional tag. "Category" was already used for one form of categorization (integration, external, long-running) and I needed another tag that distinguishes "integration"-style tests from unit tests.

I probably should have used better names but TBH I don't have any sensible counterproposals.